Form

MicroverseNelson opened the pull request:
**CHANGES**
- Added- form to the project according to template 2
- Changed card links to use flex box instead of floats
- Made changes to clear my linters.
- Added form input attributes : character length, name and required.
- Adjusted css form-imputs to accomodate styling.
- configured formspree to capture data from my form.
- changed the Button size and width and padding
- Changed the max length on the form input fields
- changed the title of this comment from chages to changes

codepantha commented on index.html:
```diff
@@ -205,10 +205,12 @@ <h2 class="contact-form-title">Get In Touch</h2>
-        <input type="text" placeholder="Full name">
-        <input type="text" placeholder="Email address">
-        <textarea name="message" id="messageID" cols="30" rows="10" placeholder="Enter text here"></textarea>
+        <form action="https://formspree.io/f/mlezozpr" method="post">
+        <input name="name" type="text" placeholder="Full name" maxlength="50" required>
+        <input name="email" type="email" placeholder="Email address" maxlength="50" required>
```
- [ ] You have set a `maxlength=50` for your inputs but the specification clearly advises `30`. Please make this change

codepantha commented on main.css:
```diff
@@ -294,34 +294,35 @@
 
-  .contact-form-section .form-inputs .getintouchbtn {
+  .contact-form-section .form-inputs form .getintouchbtn {
     background-color: #ff6b00;
     width: 80%;
     align-self: center;
-    padding: 20px;
+    padding: 16px;
```
- [ ] Let's try to follow the design guidelines for our submit button. Please adjust the `width` , `font-size`, and `font-weight` to match the Figma template 🙏 
### Figma Design
![image](https://user-images.githubusercontent.com/46839548/166437924-691012f7-ed9a-4ef7-ae98-24490b96a20f.png)
### Your Design
![image](https://user-images.githubusercontent.com/46839548/166437972-05317993-c7bc-4737-9315-ab4b42a4d5d1.png)

codepantha commented on index.html:
```diff
@@ -205,10 +205,12 @@ <h2 class="contact-form-title">Get In Touch</h2>
+        <form action="https://formspree.io/f/mlezozpr" method="post">
+        <input name="name" type="text" placeholder="Full name" maxlength="50" required>
+        <input name="email" type="email" placeholder="Email address" maxlength="50" required>
+        <textarea name="message" id="messageID" cols="30" rows="10" placeholder="Enter text here" maxlength="500" required></textarea>
         <button type="submit" class="getintouchbtn">Get in touch</button>
+      </form>
       </div>
     </section>
```
- [ ] Please adjust the `height`, the `border`, and the `background-color` of the `textarea `to match the Figma design attached below 👇 
### Figma Design
![image](https://user-images.githubusercontent.com/46839548/166438126-b3efcbad-37f0-4893-b2ee-d3e6f3d8c531.png)
### Your Design
![image](https://user-images.githubusercontent.com/46839548/166438173-21d58def-6cce-4762-bd36-bd0179d61143.png)

codepantha reviewed (changes requested):
Hi @MicroverseNelson 👋,

## STATUS: REQUESTED CHANGES ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media1.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif"/>

### Highlights 🎯 
- Linter checks ✔️ 
- Meaningful commit messages ✔️ 
- Professional README ✔️ 
- PR description ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] making your PR title a bit more descriptive.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

MicroverseNelson commented:
CHANGES

Added- form to the project according to template 2
Changed card links to use flex box instead of floats
Made changes to clear my linters.
Added form input attributes : character length, name and required.
Adjusted css form-imputs to accomodate styling.
configured formspree to capture data from my form.
changed the Button size and width and padding
Changed the max length on the form input fields
changed the title of this comment from chages to changes

Mre55 commented on main.css:
```diff
@@ -294,38 +294,42 @@
     font-size: 14px;
+    background-color: #fbf8f7;
+    width: 327px;
+    height: 114px;
```
Nice job 👍 implementing the required changes by the previous reviewer with `height` and `background-color` but one thing you missed is the `border`. Kindly, add `border: none` to stick to the Figma design.

Mre55 reviewed (changes requested):
**Hi @MicroverseNelson,**

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 

 ## Required Changes ♻️
_Check the comments under the review._

<img width="430" src="https://media1.giphy.com/media/oVjOd9TEZDmSTc4x37/giphy.gif"/>

## Optional suggestions 🟡 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

jorgegoco commented:
**changes**
Fix the contact button border.

KDlamini reviewed (approved):
### Hi @MicroverseNelson 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the required changes. 🎊 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

## HIGHLIGHTS 🌟 

✔️ Great PR title
✔️ Good PR description
✔️ Good commits
✔️ Correct use of GitFlow
✔️ No linters errors

# Status: Approved ✔️ 
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
